### PR TITLE
Run voltage init

### PR DESCRIPTION
**Pull request by etiennehomer**

### CODE REVIEW

File: src/main/java/org/gridsuite/study/server/StudyController.java [hunk 1]
```diff
@@ -776,6 +779,50 @@ public ResponseEntity<String> getShortCircuitAnalysisStatus(@Parameter(descripti
                 ResponseEntity.noContent().build();
     }
 
+    @PutMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/voltage-init/run")
+    @Operation(summary = "run voltage init on study")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The voltage init has started")})
```
Comment (AbdelHedhili) [inline src/main/java/org/gridsuite/study/server/StudyController.java:784]: not allowed is also a possible return

File: src/main/java/org/gridsuite/study/server/StudyController.java [hunk 2]
```diff
@@ -776,6 +779,50 @@ public ResponseEntity<String> getShortCircuitAnalysisStatus(@Parameter(descripti
...
+    public ResponseEntity<Void> stopVoltageInit(@Parameter(description = "Study uuid") @PathVariable("studyUuid") UUID studyUuid,
+                                                         @Parameter(description = "nodeUuid") @PathVariable("nodeUuid") UUID nodeUuid) {
+        voltageInitService.stopVoltageInit(studyUuid, nodeUuid);
+        return ResponseEntity.ok().build();
+    }
+
+    @GetMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/voltage-init/result")
+    @Operation(summary = "Get a voltage init result on study")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The voltage init result"),
+            @ApiResponse(responseCode = "204", description = "No voltage init has been done yet"),
```
Comment (AbdelHedhili) [inline src/main/java/org/gridsuite/study/server/StudyController.java:805]: has NOT been not

File: src/main/java/org/gridsuite/study/server/StudyController.java [hunk 3]
```diff
@@ -776,6 +779,50 @@ public ResponseEntity<String> getShortCircuitAnalysisStatus(@Parameter(descripti
...
+                                                        @Parameter(description = "nodeUuid") @PathVariable("nodeUuid") UUID nodeUuid) {
+        String result = voltageInitService.getVoltageInitResult(nodeUuid);
+        return result != null ? ResponseEntity.ok().body(result) :
+                ResponseEntity.noContent().build();
+    }
+
+    @GetMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/voltage-init/status")
+    @Operation(summary = "Get the voltage init status on study")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The voltage init status"),
+            @ApiResponse(responseCode = "204", description = "No voltage init has been done yet"),
```
Comment (AbdelHedhili) [inline src/main/java/org/gridsuite/study/server/StudyController.java:817]: has NOT

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 4]
```diff
@@ -127,3 +127,8 @@ databaseChangeLog:
   - include:
       file: changesets/changelog_2023-04-17T13:13:34Z.xml
       relativeToChangelogFile: true
+
+  - include:
+      file: changesets/changelog_20230428T120005Z.xml
```
Comment (AbdelHedhili) [inline src/main/resources/db/changelog/db.changelog-master.yaml:132]: Doesn't matter but the file name is not formatted like the others and it's ugly so change it if you feel like it

**Review by AbdelHedhili — APPROVED** [review]
littke doc fixes and it's ok

**Review by Meklo — APPROVED** [review]
Code : OK
Tests : Couldn't try out the feature in the current state

**Comment by etiennehomer** [comment]
> little doc fixes and it's ok
> 
> edit: after updating with main the CI fails in the test ` java.lang.AssertionError: Should not be any messages in queue study.update : expected null, but was:<GenericMessage [payload=byte[0], headers={studyUuid=ca149784-35a4-46e8-b95e-fe8d65286921, movedNode=fea24039-1814-40dc-ba25-c8e1b81c0d42, parentNode=a41ada86-8692-4265-adf8-c36bd4754d71, id=7407ba8f-471e-47e3-119a-016bdf5e6a1f, contentType=application/json, updateType=subtreeMoved, target-protocol=kafka, timestamp=1683713740699}]>`

Fixed

File: src/main/java/org/gridsuite/study/server/StudyController.java (hunk 1 above)
Comment (etiennehomer) [reply src/main/java/org/gridsuite/study/server/StudyController.java:784]: Done

File: src/main/java/org/gridsuite/study/server/StudyController.java (hunk 3 above)
Comment (etiennehomer) [reply src/main/java/org/gridsuite/study/server/StudyController.java:818]: There's the negation with the no

File: src/main/java/org/gridsuite/study/server/StudyController.java (hunk 2 above)
Comment (etiennehomer) [reply src/main/java/org/gridsuite/study/server/StudyController.java:806]: There's the negation with the no

Comment (AbdelHedhili) [reply src/main/java/org/gridsuite/study/server/StudyController.java:806]: oh mb i'm blind

File: src/main/resources/db/changelog/db.changelog-master.yaml (hunk 4 above)
Comment (etiennehomer) [reply src/main/resources/db/changelog/db.changelog-master.yaml]: Yes, it's better to keep it homogeneous and then we'll do the PR to change all the file names.
So done

**Review by AbdelHedhili — APPROVED** [review]

**Pull request merged**